tests/testthat/data-citation/encoding/inst/CITATION

```diff
@@ -0,0 +1,14 @@
+citHeader("To cite bigsnpr in publications, please use:")
+
+citEntry(entry = "Article",
```

Comment:
Similarly, here it would be useful to reduce the citation to something like `title = "title"`